aries-site/src/examples/components/dateinput/DateInputSimpleExample.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Box, DateInput } from 'grommet';
+
+export const DateInputSimpleExample = () => {
+  const [value, setValue] = React.useState('');
+  const onChange = event => {
+    const nextValue = event.value;
+    console.log('onChange', nextValue);
+    setValue(nextValue);
+  };
+  return (
+    <Box align="center" pad="large">
+      <DateInput
+        name="dateinput"
```

Comment:
I think we need an A11yTitle / aria-label here:
![Screen Shot 2020-08-12 at 12 06 36 PM](https://user-images.githubusercontent.com/1756948/90051127-7128ef00-dc94-11ea-80ea-ec0e960adb83.png)